packages/template-retail-react-app/package.json

```
@@ -9,9 +9,9 @@
     "analyze-build": "cross-env MOBIFY_ANALYZE=true npm run build",
     "build": "npm run build-translations && pwa-kit-dev build",
     "build-translations": "npm run extract-default-translations && npm run compile-translations",
-    "compile-translations": "formatjs compile-folder --ast app/translations app/translations/compiled",
-    "compile-translations:pseudo": "formatjs compile --ast app/translations/en-US.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
-    "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/en-US.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
+    "compile-translations": "formatjs compile-folder --ast translations translations/compiled",
+    "compile-translations:pseudo": "formatjs compile --ast translations/en-US.json --out-file translations/compiled/en-XB.json --pseudo-locale en-XB",
+    "extract-default-translations": "node ./scripts/extract-default-messages.js",
```

Let's have our script be able to receive any given locale. So that developers do not need to go into the script's implementation to replace `en-US` with another locale. They can simply update the default locale in this package.json file instead.

```suggestion
    "extract-default-translations": "node ./scripts/extract-default-messages.js en-US",
```

---

Good idea, I've added to include locale as an arg into the script